src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs

```diff
@@ -800,11 +804,13 @@ bool TryMatchSubscription(string topicName, DateTime messageTime, out QualityOfS
                     return;
                 }
 
+                CommonEventSource.Log.Info("ClientAuthenticated", this.identity.ToString(), this.ChannelId);
```

Comment:
Looks like String Interpolation is used all over place with the IDeviceIdentity object.